assignment.js

```diff
@@ -112,6 +145,74 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+console.log('')
+let playerAChoice = Math.random()
+let playerBChoice = Math.random()
+
+
+console.log(playerAChoice)
+//generates playerA's decsion
+ if (playerAChoice <0.333){
+     playerAChoice = "rock"
+ }else if(playerAChoice <=0.666){
+     playerAChoice = "paper"
+ }
+ else{
+     playerAChoice = "scissors"
+ }
+
+
+//generates playerB's decsion
+if (playerBChoice <0.333){
+    playerBChoice = "rock"
+}else if(playerBChoice <=0.666){
+    playerBChoice = "paper"
+}
+else{
+    playerBChoice = "scissors"
+}
+
+
+console.log(`Player One chooses ${playerAChoice} and Player Two chooses ${playerBChoice}.`)
+
+
+
+//function for winning conditions
+let compare = function(choice1,choice2){
+    if(choice1===choice2){
+        return "The result is a tie!";
+    }
+    if(choice1==="rock"){
+        if(choice2==="scissors"){
+            return "Player One wins"
+        }
+        else{
+            return "Player Two wins";
+        }
+    }
+    if(choice1==="paper"){
+        if(choice2==="rock"){
+            return "Player One wins";
+        }
+        else{
+            return "Player Two wins";
+        }
+    }
+    if(choice1==="scissors"){
+        if(choice2==="rock"){
+            return "Player Two wins";
+        }
+        else{
+            return "Player One wins";
+        }
+    }
+};
+console.log('')
+// calls compare function with the players choice
+console.log(compare(playerAChoice,playerBChoice))
+
```

Comment:
@jct117 Great job utilizing a function to evaluate the winner of the game. Take a look at this alternate approach
 [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L128)